ui/packages/catalog-ui-search/src/main/webapp/component/list-interactions/list-interactions.view.js

```diff
@@ -72,9 +77,40 @@ module.exports = Marionette.ItemView.extend({
     const newList = new this.model.constructor(copyAttributes)
     this.model.collection.add(newList)
   },
-  triggerAction(event) {
+  async triggerAction(event) {
     const url = event.currentTarget.getAttribute('data-url')
-    window.open(url)
+
+    try {
+      let res = await fetch(url, { Accept: 'application/json' })
+      if (!res.ok) {
+        throw Error(res.statusText)
+      }
+
+      if (res.headers.get('content-disposition') != null) {
+        const data = await res.blob()
+        const contentType = res.headers.get('content-type')
+        const name = getFilenameFromContentDisposition(
+          res.headers.get('content-disposition')
+        )
+        saveFile(name, contentType, data)
+      } else {
+        res = await res.json()
+      }
+
+      if (res !== undefined) {
+        announcement.announce({
+          title: 'Success',
+          message: res.message,
```

Comment:
What is the announcement message in the case that a file is returned as an attachment?

Reply:
There is no announcement message, it just downloads the file.